docs/Development/ReleaseNotes.md

```diff
@@ -30,16 +30,16 @@ This is a major update from VIC 4. The VIC 5.0.0 release aims to have nearly ide
 
 	The Classic Driver provides similar functionality as VIC 4, including ASCII and binary I/O, and a time-before-space evaluation loop order. The Classic Driver is maintained for two main reasons:
 
-		1. to provide some level of backward compatibility for existing VIC users that wish to continue using VIC using a traditional approach, and,
-		2. to allow VIC to be run at individual grid cells, without requiring the infrastructure needed by the Image Driver. Classic Driver specific documentation can be found [here](http://vic.readthedocs.org/en/develop/Documentation/Drivers/Classic/ClassicDriver/).
+	1. to provide some level of backward compatibility for existing VIC users that wish to continue using VIC using a traditional approach, and,
+	2. to allow VIC to be run at individual grid cells, without requiring the infrastructure needed by the Image Driver. Documentation for the Classic Driver can be found [here](../Documentation/Drivers/Classic/ClassicDriver/).
```

Comment:
Did you build the docs to check this link? When I build the docs with `mkdocs`, the link appears to be correct.